apps/web/src/contexts/payment-context.tsx

```diff
@@ -466,7 +466,7 @@ export function usePaymentProvider({
         if (
           !Environment.isWireEnabled ||
           (Environment.isWireEnabled &&
-            !isGreaterThanOrEqual(bid, maximumBidForCardPayments))
+            !isGreaterThan(bid, maximumBidForCardPayments))
```

Comment:
Noticed an error where if you make a bid over $2999 it wouldn't go through, since the card form is still displayed. This allows a $3000 bid, and then all future bids the bank account form will display. The bid needs to be placed though, it doesn't change the form depending on the bid amount, so this should be updated.

Reply:
This is just a temporary solution to allow an exact bid of 3000. We need the form displayed in the checkout template to change depending on what bid is placed. If the bid is over 3k, unless there was already a bid for 3k and therefore the current bid has hit that amount, currently the form will still submit as a credit card payment flow, but fail since the correct fields weren't submitted. I added more details [here](https://github.com/deptagency/algomart/issues/125). I started trying to implement, and could not get the checkout template to update. I figured this may be worth further discussion since I'm not completely sure how it should behave, but would appreciate help on how to accomplish at that point.